sygnal/webpushpushkin.py

```diff
@@ -173,6 +175,17 @@ async def _dispatch_notification_unlimited(self, n, device, context):
         payload = WebpushPushkin._build_payload(n, device)
         data = json.dumps(payload)
 
+        # web push only supports normal and low priority, so assume normal if absent
+        low_priority = n.prio == "low"
+        # allow dropping earlier notifications in the same room if requested
+        topic = None
+        if n.room_id and device.data.get("only_last_per_room") is True:
```

Comment:
```suggestion
        if n.room_id and device.data.get("only_last_per_room") == True:
```

Reply:
Actually, CI tells me it prefers the former way:
```
sygnal/webpushpushkin.py:182:64: E712 comparison to True should be 'if cond is True:' or 'if cond:'
```
So I'll revert?

Reply:
huh. my bad, sorry.